sphinx/ext/autodoc/mock.py

```diff
@@ -129,6 +129,14 @@ def invalidate_caches(self) -> None:
             sys.modules.pop(modname, None)
 
 
+def _method_is_bound(method: MethodType) -> bool:
```

Comment:
I feel `isboundmethod()` is better for this. And it's much better to move this to `sphinx.util.inspect` module.

Reply:
Done.